src/internet.ts

```diff
@@ -426,4 +426,26 @@ export class Internet {
     };
     return _password(len, memorable, pattern, prefix);
   }
+
+  /**
+   * Generates a random emoji.
+   *
+   * @param filters A list of the emoji groups that should be used.
+   *
+   * @param groups
+   * @example
+   * faker.internet.emoji() // '🥰'
+   * faker.internet.emoji(['food', 'nature']) // '🥐'
+   */
+  emoji(groups?: Array<EmojiGroup>): string {
+    groups =
+      groups ||
+      (Object.keys(this.faker.definitions.internet.emoji) as Array<
+        keyof InternetDefinitions['emoji']
+      >);
```

Comment:
```suggestion
      (Object.keys(this.faker.definitions.internet.emoji) as Array<EmojiGroup>);
```

is the cast needed at all?

Reply:
Yes, this is needed:
![image](https://user-images.githubusercontent.com/4563971/162626951-3888e7f1-3c2d-42b4-85de-39416c6e55b6.png)

Though, I updated the type in https://github.com/faker-js/faker/pull/504/commits/faba4db059a401d45c7521ca76931e94aa6eeaef